test/ussd_health_worker.test.js

```diff
@@ -177,7 +176,6 @@ describe("familyconnect health worker app", function() {
                     .setup.user.addr('0820000111')
                     .inputs(
                         {session_event: 'new'}  // dial in
-                        , '12345'  // state_auth_code - personnel code
```

Comment:
Has this state been removed somewhere or something?
I understood that you didn't need it in the previous PR because the number was recognised but this test specifies that it isn't recognised so I'm a bit confused.